app/src/main/res/layout/activity_forumn.xml

```diff
@@ -23,24 +23,20 @@
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/chat_recycler"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_marginTop="40dp"
-        android:layout_marginBottom="40dp"
-        app:layout_constraintBottom_toTopOf="@id/message_input"
+        android:layout_width="414dp"
+        android:layout_height="610dp"
```

Comment:
@Pranav915  pls revert these layout width and height changes

Reply:
Ok